plugins/Makefile.mk

```diff
@@ -23,3 +23,8 @@ include plugins/dmx4linux/Makefile.mk
 include plugins/e131/Makefile.mk
 include plugins/uartdmx/Makefile.mk
 endif
+
+dist_noinst_SCRIPTS += plugins/convert_README_to_header.sh
+
+plugins/%/PluginDescription.h: plugins/%/README.md plugins/convert_README_to_header.sh plugins/Makefile.mk
```

Comment:
Ok, that's a good point I had not thought about yet. I guess we should then just go with the easiest way to prevent that error.